src/pages/FeedbackPage.vue

```diff
@@ -10,19 +10,9 @@
         <a href="https://wiki.creativecommons.org/wiki/Slack#How_to_join_Slack">CC Slack</a>.
       </p>
       <p>
-        To report a bug, <a href="#report">click here</a>.
+        To give Feedback, <a href="https://docs.google.com/forms/d/e/1FAIpQLSfb_6yq2Md0v6S-XzsyT7p1QVhqr7MWHqInKdyYh4ReaWn4FQ/viewform?embedded=true" target="_blank">click here</a>.
```

Comment:
Please undo these changes. These are separate things and we need the Google Doc iframe

Reply:
@brenoferreira okay

Reply:
@brenoferreira But making google form to another tab will make it a much better view wise